src/components/QuranReader/EndOfScrollingControls/ChapterControls.tsx

```diff
@@ -10,16 +10,23 @@ import useScrollToTop from 'src/hooks/useScrollToTop';
 import { isFirstSurah, isLastSurah } from 'src/utils/chapter';
 import { logButtonClick } from 'src/utils/eventLogger';
 import { getSurahNavigationUrl } from 'src/utils/navigation';
+import { VersesResponse } from 'types/ApiResponses';
 import Verse from 'types/Verse';
 
 interface Props {
   lastVerse: Verse;
+  initialData: VersesResponse;
 }
 
-const ChapterControls: React.FC<Props> = ({ lastVerse }) => {
+const ChapterControls: React.FC<Props> = ({ initialData }) => {
   const { t } = useTranslation('quran-reader');
   const scrollToTop = useScrollToTop();
-  const { chapterId } = lastVerse;
+  // due to some errors this commented code don't compute right chapter id
+  // const { chapterId } = lastVerse;
+  // const chapterNumber = Number(chapterId);
+  const chaptterIdAndLastVerse = initialData.pagesLookup.lookupRange.to;
```

Comment:
There is a type the variable should be `chapterIdAndLastVerse` instead of `chaptterIdAndLastVerse`.